lib/ansible/module_utils/network/ios/facts/legacy/base.py

```diff
@@ -65,6 +66,11 @@ def parse_serialnum(self, data):
         if match:
             return match.group(1)
 
+    def parse_uptime(self, data):
+        match = re.search(r'uptime is (\S+)', data)
```

Comment:
this regex won't work as it will only match the first entry without whitespace after it.
means if you have a string "uptime is 1 week, 1 day, 23 hours, 31 minutes" it would only match the first "1".

suggestion: change the line like that
`match = re.search(r'uptime is (.+)', data)`